change/@ni-nimble-components-dcbb4630-7675-4f1f-b6cd-7c7698e74598.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit: please update the PR title to clarify that this is an HLD, not an implementation